setup.py

```diff
@@ -20,9 +20,10 @@
     else:
         raise RuntimeError('Cannot determine package version.')
 
+
 tests = ['pytest', 'pytest-cov', 'pytest-mock', 'flake8', 'mypy']
 docs = ['sphinx', 'sphinx_rtd_theme', 'sphinx-autodoc-typehints']
-features = ['GitPython', 'matplotlib']
+features = ['GitPython', 'matplotlib', 'jinja2', 'psyclone==2.1.0']
```

Comment:
Iirc, we couldn't use >= in the past, a long time ago, because the more recent version was broken/incompatible in some way that I can't remember. I've searched the team chat but can't find mention of it. It's certainly worth going through all the deps and trying the latest versions of things at some point though.